### PR TITLE
feat(scripts): additional assertions on the validate script

### PR DESCRIPTION
**Pull request by Reinis-FRP**
**Motivation**

Some of assertions were missing in DVMv2 upgrade test scripts.

**Summary**

Add additional assertions to the DVMv2 upgrade validation script.

**Details**

Added following assertions:
- `Proposer` role is correctly set in `GovernorV2`
- `previousVotingContract` variable correctly set in `VotingV2`
- `Minter` role is correctly added to new voting contract in `VotingToken`

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [x]  Untested

### CODE REVIEW

**Review by md0x — APPROVED** [review]
Perfect!

File: packages/scripts/src/upgrade-tests/voting2/2_Verify.ts [hunk 1]
```diff
@@ -53,7 +55,7 @@ async function main() {
   const oldVoting = await getContractInstance<VotingEthers>("Voting", process.env[OLD_CONTRACTS.voting]);
   const proposer = await getContractInstance<ProposerEthers>("Proposer", process.env[OLD_CONTRACTS.proposer]);
 
-  const votingV2 = await getContractInstance<VotingEthers>("Voting", process.env[NEW_CONTRACTS.voting]);
```
Comment (chrismaree) [inline packages/scripts/src/upgrade-tests/voting2/2_Verify.ts:56]: this did not affect anything due to the interfaces checked here being the same, right?

**Review by chrismaree — APPROVED** [review]

File: packages/scripts/src/upgrade-tests/voting2/2_Verify.ts (hunk 1 above)
Comment (Reinis-FRP) [reply packages/scripts/src/upgrade-tests/voting2/2_Verify.ts:56]: Previously interfaces checked were the same, but now as we add check on `previousVotingContract` that is available only in v2 we need `VotingV2` interface.

File: packages/scripts/src/upgrade-tests/voting2/2_Verify.ts [hunk 2]
```diff
@@ -53,7 +55,7 @@ async function main() {
...
+  const votingV2 = await getContractInstance<VotingV2Ethers>("VotingV2", process.env[NEW_CONTRACTS.voting]);
```
Comment (mrice32) [inline packages/scripts/src/upgrade-tests/voting2/2_Verify.ts:58]: nice catch!

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**